scripts/ui-test.sh

```diff
@@ -7,4 +7,5 @@ source ~/.bash_profile
 nvm install node
 unset NPM_CONFIG_PREFIX
 pip install tox
+make -f Makefile-docker ui-tests
```

Comment:
That command is being run inside the `tox` environment as well, so it's duplicated?

Reply:
This script is no longer used, I'll have to remove it.

Reply:
Great! Let's do this denn 😆